src/plugins/data/public/ui/search_bar/search_bar.tsx

```diff
@@ -363,6 +353,20 @@ class SearchBarUI extends Component<SearchBarProps, State> {
     }
   }
 
+  public onClearSavedQuery = () => {
```

Comment:
In general, a default behavior for `onClearSavedQuery` is already implemented in the stateful search bar.
There's no need to add it here